### PR TITLE
Feature/add abstraction

### PR DESCRIPTION
**Pull request by TeyimPila**
references #1 
requesting to merge the feature/add-abstraction, created to add inheritance, into feature/add-model

### CODE REVIEW

**Comment by TeyimPila** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/TeyimPila/tela-mis/pull-request/25/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=TeyimPila/tela-mis)

File: tela/models.py [hunk 1]
```diff
@@ -2,7 +2,28 @@
 from django.utils import timezone
 
 
-class Beneficiary(models.Model):
+class People(models.Model):
```
Comment (skaex) [inline tela/models.py]: `class People` sounds incorrect. Try `human` or `person`.

**Comment by TeyimPila** [comment]
ok

**Comment by skaex** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/TeyimPila/tela-mis/pull-request/25/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=TeyimPila/tela-mis)

**Pull request merged**